Firestore/Source/API/FIRTimestamp+Internal.h

```diff
@@ -14,17 +14,11 @@
  * limitations under the License.
  */
 
-#import <Foundation/Foundation.h>
+#import "FIRTimestamp.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
-/**
- * An FSTTimestamp represents an absolute time from the backend at up to nanosecond precision.
- * An FSTTimestamp is represented in terms of UTC and does not have an associated timezone.
- */
-@interface FSTTimestamp : NSObject <NSCopying>
-
-- (instancetype)init NS_UNAVAILABLE;
+@interface FIRTimestamp ()
```

Comment:
I'm sort of surprised declaring this separately like this works and generates no warnings. It's not something we've commonly done.